### DISCUSSION TITLE
Targeted attack

### DISCUSSION THREAD

**Discussion by Metztli** [mastodon/mastodon#23351] [General]
[This is paste: micropost_log](https://pastebin.com/dsM5pySa) of a recent attack on a Toot which obviously was not of their liking.

Wonder if there had been no ModSecurity defense would the attack had succeeded against Mastodon v3.5.3.

Banned and reported offending IP; notwithstanding, from that particular hosting provider I have seen other attacks, SSH and Web, yet complaint emails are not acknowledged.

IPs coming from IP blocks under the control of that hosting provider should be treated with caution.

**Comment by ClearlyClaire** [comment]
Looks like a purely automated attempt at various injection attacks (SQLi, XSS, …), from a quick look at the attack parameters they don't seem to be based on any knowledge of the code and would have no chance of succeeding any way. There is currently no known SQLi or XSS vulnerability in Mastodon, although of course there could be unknown ones.

**Reply by Metztli** [reply]
Your insight is appreciated, @ClearlyClaire.

Thank you.
